.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ parameters:
     default: "owner/repo"
   git_release_tag:
     type: string
-    default: v10\.[0-9]*\.[0-9]*$
+    default: v[0-9]+\.[0-9]+\.[0-9]+[\S]*$
```

Comment:
nice fix
maybe add only allowed words?
because such tag also will work `v10.4.4-_+beta..rc-123`

Reply:
I think we shouldn't limit the string after the version name, as we may want to introduce other variants of version name, e.g. alpha. 
And matching the initial part of the version name also matches the previous behavior.

Reply:
Do we need to add regex for the starting part `v10.` ? It'll match `v0.` for example.

Reply:
I guess it's our responsibility to cut the release with correct version name? I think the expression here is primarily used for distinguishing the release tags from the other tags we might create?

Reply:
Yeah, however why allow the incorrect and broad regex patterns? E.g. would `v10\.[0-9]+\.[0-9]+[\S]*$` be more strict and better?

Reply:
We will eventually have version 11 and so on, it's easy to miss the ci config when releasing new versions?

Reply:
Hmm, from this point of view... Not sure that we won't change CI on the major release update, but ok, on with leaving as is